src/Gnosis.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+library Gnosis {
+
```

Comment:
I would include the AMB (L1 and L2) and L2 Executor (admin). I would also include the tokens added to the market.

Reply:
Added AMBs and executor

Reply:
Added AMBs, tokens, and reserve tokens